fe1-web/src/features/notification/screens/NotificationScreen.tsx

```diff
@@ -0,0 +1,186 @@
+import { useNavigation } from '@react-navigation/core';
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useMemo } from 'react';
+import { FlatList, StyleSheet, Text, TouchableOpacity, View, ViewStyle } from 'react-native';
+import { Button } from 'react-native-elements';
+import { useSelector } from 'react-redux';
+
+import ScreenWrapper from 'core/components/ScreenWrapper';
+import { dispatch } from 'core/redux';
+import { Typography } from 'core/styles';
+import STRINGS from 'resources/strings';
+
+import { NotificationHooks } from '../hooks';
+import { NotificationStackParamList } from '../navigation/NotificationStackParamList';
+import {
+  discardNotifications,
+  makeReadNotificationsSelector,
+  makeUnreadNotificationsSelector,
+  NotificationState,
+} from '../reducer';
+
+interface ListSeparatorItem {
+  key: string;
+  title: string;
+}
+
+interface NotificationItem extends NotificationState {
+  key: string;
+  isLastItem: boolean;
+}
+
+const NotificationScreenStyles = StyleSheet.create({
+  notificationItem: {
+    paddingTop: 16,
+    paddingBottom: 16,
+    borderBottomColor: '#000',
+    borderBottomWidth: 1,
+  } as ViewStyle,
+
+  lastNotificationItem: {
+    borderBottomWidth: 0,
+  } as ViewStyle,
+});
+
+type NavigationProps = StackScreenProps<
+  NotificationStackParamList,
+  typeof STRINGS.notification_navigation_tab_notifications
+>;
+
+/**
+ * Renders a single item in the list of notifications which can either be
+ * a notification or a heading separating different sets of notifications
+ */
+const NotificationScreenListItem = ({ item }: { item: ListSeparatorItem | NotificationItem }) => {
+  const navigation = useNavigation<NavigationProps['navigation']>();
+
+  if ('id' in item) {
+    // notification
+    return (
+      <TouchableOpacity
+        onPress={() =>
+          navigation.navigate<'Notification'>(
+            STRINGS.notification_navigation_tab_single_notification,
+            { notificationId: item.id },
+          )
+        }>
+        <View
+          style={
+            item.isLastItem
+              ? [
+                  NotificationScreenStyles.notificationItem,
+                  NotificationScreenStyles.lastNotificationItem,
+                ]
+              : NotificationScreenStyles.notificationItem
+          }>
+          <Text>{item.title}</Text>
+        </View>
+      </TouchableOpacity>
+    );
+  }
+  // separator
+  return <Text style={Typography.important}>{item.title}</Text>;
+};
+
+/**
+ * The notification screen component displaying the list of read and unread notifications
+ */
+const NotificationScreen = () => {
+  const laoId = NotificationHooks.useCurrentLaoId();
+  const selectUnreadNotifications = useMemo(
+    () => makeUnreadNotificationsSelector(laoId.valueOf()),
+    [laoId],
+  );
+  const selectReadNotifications = useMemo(
+    () => makeReadNotificationsSelector(laoId.valueOf()),
+    [laoId],
+  );
```

Comment:
True but `makeReadNotificationsSelector` is not a selector itself, it is a function that returns a selector 😅 And if I understood correctly, the same selector reference should be used for memoization to work, i.e.

`useSelector(makeReadNotificationsSelector(laoId.valueOf()))` would not work because the selector is a new one every time.

https://redux.js.org/usage/deriving-data-selectors#creating-unique-selector-instances